demisto_sdk/commands/generate_docs/tests/test_files/mirroring_test_markdow

```diff
@@ -0,0 +1,23 @@
+## Incident Mirroring
+
+You can enable incident mirroring between Cortex XSOAR incidents and CrowdStrike Falcon %%incident type%% (available from Cortex XSOAR version 6.0.0).
+To setup the mirroring follow these instructions:
+1. Navigate to **Settings** > **Integrations** > **Servers & Services**.
+2. Search for CrowdStrike Falcon and select your integration instance.
+3. Enable Fetches incidents.
```

Comment:
I think that at this point of the readme we can replace this with something like:
```suggestion
1. Enable Fetching incidents in your instance configuration.
```